*** UpdateInfo.cmake.orig	2021-05-18 10:29:07.000000000 +0200
--- UpdateInfo.cmake	2021-05-18 10:29:22.000000000 +0200
***************
*** 96,102 ****
      include("${PROJECT_SOURCE_DIR}/ReleaseInfo.cmake")
  endif()
  
! if(WIN32)
      if(BIT_DEPTH EQUAL 4)
          set(BUILD_BIT_DEPTH 32)
          # 32 bits builds has to be installable on 64 bits system, to support WinXP/64.
--- 96,102 ----
      include("${PROJECT_SOURCE_DIR}/ReleaseInfo.cmake")
  endif()
  
! #if(WIN32)
      if(BIT_DEPTH EQUAL 4)
          set(BUILD_BIT_DEPTH 32)
          # 32 bits builds has to be installable on 64 bits system, to support WinXP/64.
***************
*** 114,120 ****
      set(SYSTEM_NAME "WinVista")
  
      configure_file("${PROJECT_SOURCE_DIR}/tools/win/InnoSetup/WindowsInnoSetup.iss.in" "${CMAKE_BINARY_DIR}/rtdata/WindowsInnoSetup.iss")
! endif()
  
  # build version.h from template
  configure_file("${PROJECT_SOURCE_DIR}/rtgui/version.h.in" "${CMAKE_BINARY_DIR}/rtgui/version.h")
--- 114,120 ----
      set(SYSTEM_NAME "WinVista")
  
      configure_file("${PROJECT_SOURCE_DIR}/tools/win/InnoSetup/WindowsInnoSetup.iss.in" "${CMAKE_BINARY_DIR}/rtdata/WindowsInnoSetup.iss")
! #endif()
  
  # build version.h from template
  configure_file("${PROJECT_SOURCE_DIR}/rtgui/version.h.in" "${CMAKE_BINARY_DIR}/rtgui/version.h")
