*** rtgui/main.cc.orig	2019-09-13 14:34:11.000000000 +0200
--- rtgui/main.cc	2019-09-13 14:31:06.000000000 +0200
***************
*** 475,480 ****
--- 475,488 ----
      }
  #endif
  
+ #ifdef WIN32
+     std::string gtk_csd = Glib::getenv("GTK_CSD");
+     if(gtk_csd.empty()) {
+       Glib::setenv("GTK_CSD", "0", true);
+     }
+ #endif
+ 
+ 
  #ifdef BUILD_BUNDLE
      char exname[512] = {0};
      Glib::ustring exePath;
